discovery_engine_core/core/src/engine.rs

```diff
@@ -108,6 +110,7 @@ pub enum Error {
 }
 
 /// Configuration settings to initialize Discovery Engine with a [`xayn_ai::ranker::Ranker`].
+#[cfg_attr(test, derive(Clone))]
```

Comment:
Any reason we don't allow this always?

Reply:
Nothing in particular, I just only needed it for the test, so that's where I added it :)

Reply:
Fixed.